jax/api.py

```diff
@@ -410,6 +410,29 @@ def papply_fun(*args, **kwargs):
 
 
 def jvp(fun, primals, tangents):
+  """Computes a (forward-mode) Jacobian-vector product of `fun`.
+
+  Args:
+    fun: Function to be differentiated. Its arguments should be arrays, scalars,
+      or standard Python containers of arrays or scalars. It should return an
+      array, scalar, or standard Python container of arrays or scalars.
+    primals: The primal values at which the Jacobian of `fun` should be
+      evaluated. Should be a tuple of arrays, scalar, or standard Python
+      container thereof.
+    tangents: The tangent vector for which the Jacobian-vector product should be
+      evaluated. Should be a tuple of arrays, scalar, or standard Python
```

Comment:
Could add that tangents should have the same shape as primals, and that the output is a tangent vector with the same shape as primals_out. 

Could also add that the length of primals and tangents is equal to the number of formal parameters to fun.